sources/packages/backend/apps/api/src/services/student-scholastic-standings/student-scholastic-standings.service.ts

```diff
@@ -136,12 +136,18 @@ export class StudentScholasticStandingsService extends RecordDataModelService<St
       // Get existing offering.
       const existingOffering = await this.offeringRepo
         .createQueryBuilder("offering")
-        .select(["offering", "educationProgram.id", "institutionLocation.id"])
+        .select([
```

Comment:
@ann-aot I can very certainly recollect that we discussed in grooming about setting parent offering id to the cloned offering. Did anything change ? I know it was not very explicit in the AC of the ticket. But I thought it was implied in `while an offering is created`

Reply:
@dheepak-aot  During scholastic standing cloning, since `parent_offering_id` was not retrieved in the offering to be cloned, the new cloned offering did not have the parent offering id, so, here the change is to retrieve `parent_offering_id`, so it will copied to the new cloned offering.
The AC had a point to check that,
![image](https://github.com/bcgov/SIMS/assets/77353155/9816fcf3-58b9-41b0-8d65-59d915a06639)

Reply:
I got it @ann-aot. Thanks for the explaination.